packages/drivers/odsp-driver-definitions/src/errors.ts

```diff
@@ -42,6 +42,11 @@ export enum OdspErrorType {
     // This condition will result in any local changes being lost (i.e. only way to save state is by user
     // copying it over manually)
     cannotCatchUp = "cannotCatchUp",
+
+    // SPO can occasionally return 403 for r/w operations on document when there is a fail over to another data center.
```

Comment:
The code comment here is too detailed about inner workings and speculating the state of ODSP's service. We should simply understand the response and handling prescribed. 

"Why" makes for great code comments, but only up until the point the commentary is dependent on private details of something else.

Reply:
Shortened the comment a bit. Just wanted a bit why it becomes r/w.